ci-operator/config/openshift/assisted-image-service/openshift-assisted-image-service-release-ocm-2.4.yaml

```diff
@@ -51,7 +42,7 @@ test_binary_build_commands: |
   go mod vendor
 tests:
 - as: mirror-nightly-image
-  cron: 41 5 */13 * *
+  cron: '@daily'
```

Comment:
this might still be affected because I didn't change OCP to be >4.9 (but it also doesn't sound right to do so just because of the script)